ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -376,8 +376,15 @@ objects:
         }
 
         function run-upgrade-tests() {
-          openshift-tests run-upgrade "${TEST_SUITE}" --to-image "${RELEASE_IMAGE_LATEST}" \
-            --provider "${TEST_PROVIDER:-}" -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
+          if [[ "${ENABLE_FIPS}" == true ]]; then
+            oc adm upgrade --to-image "${RELEASE_IMAGE_LATEST}" --allow-explicit-upgrade --force
+            fips_check
```

Comment:
testing that ^^ wip

Reply:
yea, that works, upgrade succeeds, but the `openshift-tests run-upgrade` fails with fips cluster. :woman_shrugging: